docs/rules/semicolon/index.html

```diff
@@ -10,6 +10,7 @@
 
```

Comment:
this file update shouldn't be in this PR, it should only happen when you're updating the docs as a whole (on release)